Disable an 'import rosgraph' console warning.

Reasoning for not upstreaming this patch: Drake-specific build option.

--- xacro/cli.py
+++ xacro/cli.py
@@ -97,7 +97,7 @@
         mappings = load_mappings(argv)
         filtered_args = [a for a in argv if REMAP not in a]  # filter-out REMAP args
     except ImportError as e:
-        warning(e)
+        # warning(e)
         mappings = {}
         filtered_args = argv
 
